From: Kamil Dudka <kdudka@redhat.com>
Date: Fri, 19 Feb 2021 14:42:04 +0100
Applied-Upstream: https://github.com/logrotate/logrotate/commit/41063425326c88ce17246d1ad5cf6c1f67a5fe10
Subject: logrotate.8: make the /var/log/news example consistent

We should use `sharedscripts` to make sure the daemon is signalled
only once.  Moreover, the note about log files not being compressed
on failure was misleading because the example uses `nocompress`.

Fixes: https://github.com/logrotate/logrotate/issues/374
---
 logrotate.8.in | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/logrotate.8.in b/logrotate.8.in
index f27c279..ef9dcdd 100644
--- a/logrotate.8.in
+++ b/logrotate.8.in
@@ -154,6 +154,7 @@ compress
     rotate 2
     olddir /var/log/news/old
     missingok
+    sharedscripts
     postrotate
         kill \-HUP $(cat /var/run/inn.pid)
     endscript
@@ -192,9 +193,7 @@ Normal shell quoting rules apply, with \fB'\fR, \fB"\fR, and \fB\e\fR
 characters supported.
 
 The next section defines the parameters for all of the files in
-\fI/var/log/news\fR. Each file is rotated on a monthly basis.  This is
-considered a single rotation directive and if errors occur for more than
-one file, the log files are not compressed.
+\fI/var/log/news\fR. Each file is rotated on a monthly basis.
 
 The last section uses tilde expansion to rotate log files in the home
 directory of the current user.  This is only available, if your glob
